src/Microsoft.AspNetCore.Server.Kestrel/Systemd/KesterlServerOptionsSystemdExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.AspNetCore.Server.Kestrel;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    public static class KesterlServerOptionsSystemdExtensions
```

Comment:
Yes, we should decide if we want it baked into the Kestrel package. Even https is a separate package.